qiskit/providers/ibmq/__init__.py

```diff
@@ -95,7 +95,7 @@
 from .ibmqbackendservice import IBMQBackendService
 from .utils.utils import setup_logger
 from .runner_result import RunnerResult
-
+from .runtime import ProviderRequestParams
```

Comment:
Since this is a runtime specific class, I would just leave it in `/ibmq/runtime/__init__.py` and not here.

Reply:
Fixed!